README.md

```diff
@@ -27,6 +27,21 @@ Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
 
 Ready to run in production? Please [check our deployment guides](https://hexdocs.pm/phoenix/deployment.html).
 
+## Building release
+
+To build a release including Erlang Real-Time Sytem ready to be deployed on a
+production server:
+
+  * Compile application with `ENV=prod mix compile`
+  * Compile assets with `npm run deploy --prefix ./assets`
+  * Generate assets digests with `ENV=prod mix phx.digest`
```

Comment:
You can regroup these 2 commands with your new alias: `ENV=prod mix compile_assets`

Reply:
that was my intention, I just forgot to update the README ;)
Thanks for spotting this out.